spec/acceptance/approval_from_web_app.feature

```diff
@@ -1,106 +0,0 @@
-    When I go to the approval_response page without a token
-    Then I should not see 'Approve'
-
-  Scenario: An approver visits the page to approve in turn
-    Given a linear proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-    And the proposal has an approval for 'supervisor2@test.gov' in position 2
-    And the proposal has been approved by 'supervisor1@test.gov'
-    And the logged in user is 'supervisor2@test.gov'
-    When I go to the approval_response page without a token
-    Then I should see 'Approve'
-    When I click 'Approve'
-    Then I should see alert text 'You have approved'
-
-  Scenario: An approver visits the page to approve out of turn
-    Given a linear proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-    And the proposal has an approval for 'supervisor2@test.gov' in position 2
-    And the logged in user is 'supervisor2@test.gov'
-    When I go to the approval_response page without a token
-    Then I should not see 'Approve'
-
-  Scenario: An approver for a parallel proposal visits the page to approve
-    Given a parallel proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-    And the proposal has an approval for 'supervisor2@test.gov' in position 2
-    And the logged in user is 'supervisor2@test.gov'
-    When I go to the approval_response page without a token
-    Then I should see 'Approve'
-    When the logged in user is 'supervisor1@test.gov'
-    And I go to the approval_response page without a token
-    Then I should see 'Approve'
-    When I click 'Approve'
-    Then I should see alert text 'You have approved'
-
-  Scenario: A requester for a parallel proposal visits the page
-    Given a parallel proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-    And the proposal has an approval for 'supervisor2@test.gov' in position 2
-    Given the logged in user is 'requester1@some-dot-gov.gov'
-    When I go to the approval_response page without a token
-    Then I should not see 'Approve'
-    When the logged in user is 'supervisor1@test.gov'
-    And I go to the approval_response page without a token
-    Then I should see 'Approve'
-    When the logged in user is 'supervisor2@test.gov'
-    And I go to the approval_response page without a token
-    Then I should see 'Approve'
-    When I click 'Approve'
-    Then I should see alert text 'You have approved'
-
-  Scenario: Displaying approval actions for a parallel proposal
-    Given a parallel proposal
-    And the proposal has an approval for 'supervisor1@test.gov' in position 1
-
-    When the logged in user is 'supervisor1@test.gov'
-    And I go to the approval_response page without a token
-    And I should see 'Approve'
-
-    When I click 'Approve'
-    Then I should see alert text 'You have approved'
-    And I should not see 'Approve'
-
-  Scenario: Displaying approval actions for a linear proposal
```

Comment:
this is covered by https://github.com/18F/C2/blob/master/spec/features/display_status_spec.rb#L36